backend-api/src/functions/common/session/updateOperations/ResultSent.ts

```diff
@@ -0,0 +1,17 @@
+import { IUpdateSessionOperation, SessionState } from "../Session";
+
+export class ResultSent implements IUpdateSessionOperation {
+  getDynamoDbUpdateExpression() {
+    return "set sessionState = :resultSent";
+  }
+
+  getDynamoDbConditionExpression(): undefined {
+    return undefined;
```

Comment:
I assume this is undefined, given this is just a spike?

Reply:
Or is this because we've already fetched the session record earlier in the lambda and we know in the lambda itself whether we want to make an update operation or not?

Reply:
The second one!